src/crypto/test/crypto.cpp

```diff
@@ -378,7 +378,6 @@ void run_csr(bool corrupt_csr = false)
 
   std::vector<SubjectAltName> subject_alternative_names;
 
-  // mbedtls doesn't support parsing SAN from CSR
   if constexpr (std::is_same_v<T, KeyPair_OpenSSL>)
```

Comment:
The `else` branch should be deletable now I believe.

Reply:
And there's no longer the need for the loop in `TEST_CASE("Create sign and verify certificates")`.

Reply:
Of course! Thanks!

Reply:
> And there's no longer the need for the loop in `TEST_CASE("Create sign and verify certificates")`.

I'm not sure I agree. This is related to corrupting the CSR and checking for the throw, not any previous mbedTLS behaviour.